api/BluetoothRemoteGATTService.json

```diff
@@ -0,0 +1,456 @@
+{
+  "api": {
+    "BluetoothRemoteGATTService": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/BluetoothRemoteGATTService",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": [
+            {
+              "version_added": "45",
+              "version_removed": "53",
+              "notes": "Behind a preference flag in Chrome OS."
+            },
+            {
+              "version_added": "51",
+              "version_removed": "53",
+              "notes": "Linux support."
+            },
+            {
+              "version_added": "53",
```

Comment:
Newest supported version should be at the top, not the bottom. This is true across the file.